packages/react/src/components/Inbox.tsx

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+import type { InboxNotification } from '@novu/js';
+import type { BaseNovuUIOptions } from '@novu/js/ui';
+import { useRenderer } from '../context/RenderContext';
+import { Mounter } from './Mounter';
+import { Renderer } from './Renderer';
+
+type InboxDefaultProps = {
+  renderNotification?: (notification: InboxNotification) => React.ReactNode;
+  renderBell?: ({ unreadCount }: { unreadCount: number }) => React.ReactNode;
+};
+
+const InboxDefault = (props: InboxDefaultProps) => {
+  const { renderNotification, renderBell } = props;
+  const { novuUI, mountElement } = useRenderer();
+
+  const mount = React.useCallback(
+    (element: HTMLElement) => {
+      return novuUI.mountComponent({
+        name: 'Inbox',
+        props: {
+          renderNotification: renderNotification
+            ? (el, { notification }) => mountElement(el, renderNotification(notification))
+            : undefined,
+          renderBell: renderBell ? (el, { unreadCount }) => mountElement(el, renderBell({ unreadCount })) : undefined,
+        },
+        element,
+      });
+    },
+    [renderNotification, renderBell]
+  );
+
+  return <Mounter mount={mount} />;
+};
+
+type BaseProps = BaseNovuUIOptions;
+
+type DefaultProps = BaseProps &
+  InboxDefaultProps & {
+    children?: never;
+  };
+
+type WithChildrenProps = BaseProps & {
+  children: React.ReactNode;
+};
```

Comment:
<!-- Chores are simple tasks that must be done before the subject can be “officially” accepted. Usually, these comments reference some common process. Try to leave a link to the process description so that the reader knows how to resolve the chore.  -->
☑ **chore:** Please use `React`'s types rather than defining our own. If we need to define our own foundational types, please move them to a separate file so that they can be imported into other files (and this one) without risk of circular dependency‏